libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/LoggingContext.scala

```diff
@@ -9,22 +9,23 @@ import org.slf4j.Marker
 
 import scala.collection.JavaConverters.mapAsJavaMapConverter
 
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
 object LoggingContext {
 
   def newLoggingContext[A](f: LoggingContext => A): A =
     f(new LoggingContext(Map.empty))
 
-  def newLoggingContext[A](kv: (String, String), kvs: (String, String)*)(
-      f: LoggingContext => A): A =
+  def newLoggingContext[A](kv: (String, Any), kvs: (String, Any)*)(f: LoggingContext => A): A =
     f(new LoggingContext((kv +: kvs).toMap))
 
-  def withEnrichedLoggingContext[A](kv: (String, String), kvs: (String, String)*)(
-      f: LoggingContext => A)(implicit logCtx: LoggingContext): A =
+  def withEnrichedLoggingContext[A](kv: (String, Any), kvs: (String, Any)*)(f: LoggingContext => A)(
```

Comment:
This means that the key can be any arbitrarily complex object, which I would recommend not doing without a good plan. I'm ok with the private `ctxMap` to have keys of type `Any` but please only add publicly-facing methods with specific signatures based on need.